src/components/NavigationLinks/NavigationLinks.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+import './NavigationLinks.scss'
+
+const NavigationLinks = (props = []) => {
+  const navLinks = props.navLinks;
+  return (
+    <>
+      {navLinks &&
+        <ul className='navigation-links'>
+          { navLinks.map((item, index) => {
```

Comment:
Add null check here `navLinks? .map((item`

Also might be better in some cases to remove the check above the `ul` element u don't want the website to look different in case of no `navLinks` found